### ISSUE TITLE
captcha / tampermonkey

### ISSUE THREAD

**Issue by UserX404** [pyload/pyload#4405]
### Who is capable/responsible for the tampermonkey skript?

Since today my captchas won't resize to it's correct (full) size as shown in the attached screenshot. So i'm not able to solve them within pyload UI.

### screenshot

![grafik](https://github.com/pyload/pyload/assets/14954521/99b3c7a9-12d5-4c58-bc43-9fe1a2c10a40)

**Comment by UserX404** [comment]
HCaptcha.
Can you elaborate which url you need? 

```
236 | 2023-12-04 20:43:05 | DEBUG | pyload | DOWNLOADER DdownloadCom[18559]: LOAD URL https://ddownload.com/a8t0lo551pb7 \| get={} \| post={} \| ref=False \| cookies=True \| just_header=False \| decode=True \| multipart=False \| redirect=True \| req=None
-- | -- | -- | -- | --
237 | 2023-12-04 20:43:05 | INFO | pyload | DOWNLOADER DdownloadCom[18559]: Checking for link errors...
238 | 2023-12-04 20:43:05 | INFO | pyload | DOWNLOADER DdownloadCom[18559]: No errors found
239 | 2023-12-04 20:43:05 | INFO | pyload | DOWNLOADER DdownloadCom[18559]: Processing as free download...
240 | 2023-12-04 20:43:05 | DEBUG | pyload | DOWNLOADER DdownloadCom[18559]: Getting download link #1...
241 | 2023-12-04 20:43:05 | INFO | pyload | DOWNLOADER DdownloadCom[18559]: Checking for link errors...
242 | 2023-12-04 20:43:05 | INFO | pyload | DOWNLOADER DdownloadCom[18559]: No errors found
243 | 2023-12-04 20:43:05 | DEBUG | pyload | DOWNLOADER DdownloadCom[18559]: {'op': 'download2', 'id': 'a8t0lo551pb7', 'rand': 'brw2wwd5n2ddss4dciuy5p6rdu746hbpltw4uffsvq', 'referer': '', 'method_free': '', 'method_premium': '', 'adblock_detected': ''}
244 | 2023-12-04 20:43:05 | DEBUG | pyload | ANTICAPTCHA DdownloadCom[18559]: ReCaptcha \| 122129ec-9e86-4ace-949e-19422b57364e \| Wrong key format, this probably because it is not a reCAPTCHA key
245 | 2023-12-04 20:43:05 | WARNING | pyload | ANTICAPTCHA DdownloadCom[18559]: ReCaptcha \| Key pattern not found
246 | 2023-12-04 20:43:05 | DEBUG | pyload | ANTICAPTCHA DdownloadCom[18559]: HCaptcha \| Key: 122129ec-9e86-4ace-949e-19422b57364e
247 | 2023-12-04 20:43:05 | DEBUG | pyload | ANTICAPTCHA DdownloadCom[18559]: HCaptcha \| Challenge hCaptcha interactive
248 | 2023-12-04 20:43:05 | DEBUG | pyload | ADDON Captcha9Kw: LOAD URL https://www.9kw.eu/index.cgi \| get={'apikey': '**REMOVED**', 'pyload': '1', 'source': 'pyload', 'action': 'usercaptchaguthaben'} \| post={} \| ref=True \| cookies=True \| just_header=False \| decode=True \| multipart=False \| redirect=True \| req=None
```

**Comment by mihawk90** [comment]
Same here.

I feel like this whole resizing business could be avoided if the dialogue window was just bigger in the first place. Not sure why it's kept so small just so that it has to be resized anyway.

But anyway, same issue here. Example-URL:
https://ddownload.com/ryr79clf2bm8

**Comment by michnixweiss** [comment]
Same problem here.

I noticed the following:
In the first step the window is resized to the "im no robot". When checking it the pictures apear, but the first one is still present - in background, I can click it at the top of the window, a small line.
Dont know if this is correct or has been before.
Hope it helps.

**Comment by mihawk90** [comment]
My temporary (or possibly permanent, because I'm tired of the resizing weirdness) workaround is a User-CSS in Stylish:

```CSS
#cap_iframe_clipping {
    overflow: auto !important;
    height: 500px !important;
    width: 500px !important;
}
```

The `!important` is to override the style-attribute on the element.

**Comment by UserX404** [comment]
Manually updated the plugin and want to report back it's resizing again!
Thanks for the quick fix. 🥳

**Comment by michnixweiss** [comment]
> My temporary (or possibly permanent, because I'm tired of the resizing weirdness) workaround is a User-CSS in Stylish:
> 
> ```CSS
> #cap_iframe_clipping {
>     overflow: auto !important;
>     height: 500px !important;
>     width: 500px !important;
> }
> ```
> 
> The `!important` is to override the style-attribute on the element.

Where do I have to insert your workaround?

**Comment by UserX404** [comment]
> > My temporary (or possibly permanent, because I'm tired of the resizing weirdness) workaround is a User-CSS in Stylish:
> > ```css
> > #cap_iframe_clipping {
> >     overflow: auto !important;
> >     height: 500px !important;
> >     width: 500px !important;
> > }
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > The `!important` is to override the style-attribute on the element.
> 
> Where do I have to insert your workaround?

You don't need a workaround. I haven't checkt if the fix is already availabe with pip or however you run pyload but you can update the file manually.
within your install path: 
```
cd /module/plugins/captcha/
wget https://raw.githubusercontent.com/pyload/pyload/9f3e5bde3fab138570fb8585325145626813b974/module/plugins/captcha/HCaptcha.py
```

**Comment by mihawk90** [comment]
> Where do I have to insert your workaround?

As I said it's a User-CSS, there's a myriad of options to apply them. Although I mistyped, because I meant [Stylus](https://en.wikipedia.org/wiki/Stylus_(browser_extension)), not Stylish.

**Comment by michnixweiss** [comment]
Thanks @UserX404 that worked for me.
Sorry @mihawk90 but I don't know what a User-CSS is, so needed some help.

**Comment by BobSilent** [comment]
@GammaC0de this needs to be merged to `main` as well for pyload 0.5
